app/javascript/src/components/Dashboard/Notes/Note/index.jsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+
+import { MenuVertical, Clock } from "neetoicons";
+import { Typography, Dropdown, Button, Avatar } from "neetoui";
+
+import { NOTES } from "../constants";
+
+const MainContent = () =>
```

Comment:
Go through the Slack thread carefully. There is a point regarding component name and filename consistency. Here calling this component `MainContent` doesn't make sense.

Reply:
Fixed the component name.